app/helpers/ops_helper.rb

```diff
@@ -101,4 +101,77 @@ def miq_summary_rbac_details(users, groups, roles, tenants)
     data[:rows] = rows
     miq_structured_list(data)
   end
+
+  def select_server_details(record, role = nil)
+    data = {:mode => "selected_server_roles"}
+    rows = []
+    rows.push(row_data(_('Server'), "#{record.name}(#{record.id})"))
+    rows.push(row_data(_('Hostname'), record[:hostname]))
+    rows.push(row_data(_('IP Address'), record[:ipaddress]))
+    rows.push(row_data(_('Status'), record[:status]))
+    rows.push(row_data(_('Process ID'), record[:pid]))
```

Comment:
this can be done as 
```
rows = [
row_data(_('Server'), "#{record.name}(#{record.id})"),
row_data(_('Hostname'), record[:hostname]),
row_data(_('IP Address'), record[:ipaddress]),
row_data(_('Status'), record[:status]),
row_data(_('Process ID'), record[:pid])
]
```